modules/hitl/src/views/full/sidebar/index.jsx

```diff
@@ -26,7 +26,19 @@ export default class Sidebar extends React.Component {
     this.props.toggleOnlyPaused()
   }
 
-  renderUser = (value) => {
+  toggleSearch = () => {
+    this.props.customSearchProps.searchClickAction()
```

Comment:
Try to keep naming clear, is it toggling or clicking ?  it could be like handleSearchClicked

Reply:
[Done](https://github.com/shamoh19/botpress/commit/3c171a8926b82c779d9f24997da72b05e98cd396#diff-aea332c672541f3ad09f68a0c3dddcd4)